### ISSUE TITLE
[ossia-max] Max window spammed with HTTP Error message when deleting model in oscquery-connected patch

### ISSUE THREAD

**Issue by evanmtp** [OSSIA/libossia#518]
I have a setup where all of my models are in one patch and my views are in another. The two patches are connected via oscquery. When I delete one of the models, about 90 lines of "HTTP Error: status code 404" get posted to the Max console. It doesn't seem to cause any performance issues or crashes, but it's alarming and clogs up the console a bit.

![Screen Shot 2019-10-11 at 5 46 05 PM](https://user-images.githubusercontent.com/5342408/66686941-04e1f700-ec4f-11e9-9c4c-d2a14c8326ea.png)

**Comment by jcelerier** [comment]
does that still happen ?

**Comment by evanmtp** [comment]
No, seems to be fine. Here are the patches I used to test just now, if you want to verify: [oscquery_test.zip](https://github.com/ossia/libossia/files/8179385/oscquery_test.zip)

Steps to test:

1. Open ossia_oscquery_test_client.maxpat and ossia_oscquery_test_server.maxpat .
2. In ossia_oscquery_test_server.maxpat , click the "expose oscquery" message box.
3. In ossia.oscquery_test_client.maxpat, click the "connect oscquery" message box.
4. In ossia_oscquery_test_server.maxpat, delete [p mySynthModel].
5. . Check the Max console for error messages.

Unfortunately, in the course of testing this, I ran afoul of #765  .
